src/components/MaColumns/MaColumns.spec.js

```diff
@@ -61,16 +61,19 @@ describe('MaColumns', () => {
     expect(console.error).not.toHaveBeenCalled()
   })
 
-  test('adds auto flow if columns has autoFlow operator', () => {
+  test('adds constant column width if columns has autoFlow operator', () => {
     const { contentWrapper } = renderComponent({ columns: '*' })
-    expect(contentWrapper).toHaveClass('has-auto-flow')
+    expect(contentWrapper).toHaveAttribute(
+      'style',
+      expect.stringContaining('1fr')
+    )
```

Comment:
just to get a bit fancy, there's `toHaveStyle`:


```suggestion
    expect(contentWrapper).toHaveStyle({ gridAutoColumns: '1fr' })
```

Reply:
nicee 🔝